aws/src/main/java/org/apache/iceberg/aws/glue/GlueTableOperations.java

```diff
@@ -194,8 +201,12 @@ void persistGlueTable(Table glueTable, Map<String, String> parameters, TableMeta
               .name(tableName)
               .tableType(GLUE_EXTERNAL_TABLE_TYPE)
               .parameters(parameters)
-              .build())
-          .build());
+              .build());
+      // Use Optimistic locking with version id while updating table
+      if (!LOAD_VERSION_ID.isNoop()) {
+        updateTableRequest.versionId(glueTable.versionId());
```

Comment:
Why not use reflection to call the method as well? That seems slightly safer, although I guess it is fine as long as you've tested this with older SDK versions.

Reply:
I agree, used reflection to invoke the method.